package.json

```diff
@@ -21,10 +21,16 @@
     "tus-js-client": "^1.5.1"
   },
   "devDependencies": {
+    "chai": "^4.x.x",
+    "mocha": "^5.x.x",
+    "nyc": "^13.x.x",
+    "sinon": "^7.x.x",
     "standard": "*"
   },
   "scripts": {
-    "standard": "standard index.js lib/*.js example/*.js",
-    "test": "npm run standard"
+    "standard": "standard index.js lib/*.js example/*.js test/**/*.js",
+    "mocha": "nyc mocha --recursive --reporter landing --timeout=3000",
+    "test": "npm run standard && npm run mocha",
+    "report-coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov"
```

Comment:
Can you add `codecov` as a dev dependency?

```
$ npm run report-coverage

> vimeo@2.1.1 report-coverage /Users/ursenbachj/code/vimeo.js
> nyc report --reporter=text-lcov > coverage.lcov && codecov

sh: codecov: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! vimeo@2.1.1 report-coverage: `nyc report --reporter=text-lcov > coverage.lcov && codecov`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the vimeo@2.1.1 report-coverage script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ursenbachj/.npm/_logs/2019-01-18T14_06_45_875Z-debug.log
```

Reply:
Done. Sorry about that.